.github/workflows/release.yml

```diff
@@ -55,3 +55,60 @@ jobs:
         uses: pypa/gh-action-pypi-publish@897895f1e160c830e369f9779632ebc134688e1b
         with:
           repository-url: https://upload.pypi.org/legacy/
+
+  publish-docker-operator:
+    needs:
+      - packages
+    runs-on: ubuntu-latest
+    permissions:
+      attestations: write
+      id-token: write
+      contents: write
+    env:
+      DOCKER_IMAGE_NAME: docker.elastic.co/observability/elastic-otel-python-operator
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@988b5a0280414f521da01fcc63a27aeeb4b104db # v3.6.1
+
+      - name: Log in to the Elastic Container registry
+        uses: docker/login-action@9780b0c442fbb1117ed29e0efdff1e18412f7567 # v3.3.0
+        with:
+          registry: ${{ secrets.ELASTIC_DOCKER_REGISTRY }}
+          username: ${{ secrets.ELASTIC_DOCKER_USERNAME }}
+          password: ${{ secrets.ELASTIC_DOCKER_PASSWORD }}
+
+      - uses: actions/download-artifact@v4
+        with:
+          name: packages
+          path: dist
+
+      - name: Extract metadata (tags, labels)
+        id: docker-meta
+        uses: docker/metadata-action@8e5442c4ef9f78752691e2d8f8d19755c6f78e81  # v5.5.1
```

Comment:
I wonder whether some metadata labels can be added:
- https://github.com/docker/metadata-action#inputs
- https://github.com/opencontainers/image-spec/blob/main/annotations.md

I guess the below ones might be relevant
```
vendor
title
description
licenses
```

Reply:
licenses would be hard since there are a ton if dependencies. Something like this?

```
diff --git a/.github/workflows/release.yml b/.github/workflows/release.yml
index f2f060c..b12dbcc 100644
--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -92,6 +92,10 @@ jobs:
           tags: |
             type=raw,value=latest,prefix=test-,enable={{is_default_branch}}
             type=semver,pattern={{version}}
+          labels:
+            org.opencontainers.image.vendor=Elastic
+            org.opencontainers.image.title=elastic-otel-python-operator
+            org.opencontainers.image.description=EDOT Python image for Kubernetes OpenTelemetry Operator
 
       - name: Build and push image
         id: docker-push
```

Reply:
Uhm title is supposed to human readable but it would something like what I have put into description

Reply:
In the image there are all the python instrumentations available and their dependencies, it's not just our own code.

Reply:
Ok, thanks for the clarification 🙇

Reply:
I updated the PR with the code above